src/ui/zcl_abapgit_html_action_utils.clas.abap

```diff
@@ -431,4 +436,11 @@ CLASS zcl_abapgit_html_action_utils IMPLEMENTATION.
 
   ENDMETHOD.
 
+  METHOD requirement_decode.
```

Comment:
looks like this method is not called from anywhere?

Reply:
Good catch, it was from an earlier approach. I removed it.